composer.json

```diff
@@ -3,7 +3,7 @@
 	"license": "MIT",
 	"require": {
 		"php": "~7.1",
-		"nette/di": "~2.4 || ~3.0",
+		"nette/di": "~3.0",
```

Comment:
I think it's better to specify here `~3.0.2 || ~3.1` instead of having the `conflict` section.

Reply:
3.1.x doesn't exist yet

Reply:
in that case just `~3.0.2` is probably fine

Reply:
I thought about it but I was not sure ithis

Reply:
`~3.0` is incorrect. It should be `~3.0.2`

Reply:
It is inrevelant because of commit 801a9b849347165107f2b450dda0b1cdc909052b . It is compatible with al version of nette di

Reply:
it looks it is compatible with 2.4, so you can revert changes in composer.json

Reply:
I reveted composer,.json